src/data.js

```diff
@@ -8,7 +8,7 @@ let Entity = require('azure-entities');
 let Namespace = Entity.configure({
   version:          1,
   partitionKey:     Entity.keys.StringKey('namespace'),
-  rowKey:           Entity.keys.StringKey('username'),
+  rowKey:           Entity.keys.ConstantKey('namespace'),
```

Comment:
Changed this to ensure an entity exists error is thrown (and subsequently caught) when an existing namespace is requested (otherwise the partition + row key would be unique every time, since username is slugid).